src/main/java/io/swagger/codegen/v3/generators/DefaultCodegenConfig.java

```diff
@@ -175,6 +176,7 @@
     protected boolean useOas2 = false;
     protected boolean copyFistAllOfProperties = false;
     protected boolean ignoreImportMapping;
+    private boolean notNullJacksonAnnotation;
```

Comment:
this is specific to a generator, not to the whole set, DefaultCodegen must be left unchanged, you would code this into the Java generator

Reply:
done